src/debug.h

```diff
@@ -197,6 +197,31 @@ void limitDebugClass( int );
  */
 bool debug_has_error_been_observed();
 
+/**
+ * Class for capturing debugmsg.
+ * Сan also use a captureDebugmsgLambda function instead.
+ */
+class captureDebugmsg
+{
+    public:
+        captureDebugmsg();
+        // @return std::string debugmsg
+        std::string dmsg();
+        ~captureDebugmsg();
```

Comment:
```suggestion
class capture_debugmsg
{
    public:
        capture_debugmsg();
        // @return std::string debugmsg
        std::string dmsg();
        ~capture_debugmsg();
```
We generally use `snake_case`.

Reply:
done